flipperdevices/Flipper-Android-App
# Refactor/remove kapt from modules

**Background**

Currently, some modules with custom Graphs and Components requires to have kapt inside them, which is blocker for KMP integration

**Changes**

- Removed kapt from non-app/non-sample modules
- Create manual implementations for DI Components
- Added `getOrCreate` for `ComponentHolder` to add self-created DI Components

**Test plan**

- Launch application
- See sync, wear service is working
